path_oidc.go

```diff
@@ -13,26 +15,31 @@ import (
 	"github.com/hashicorp/go-uuid"
 	"github.com/hashicorp/vault/sdk/framework"
 	"github.com/hashicorp/vault/sdk/helper/cidrutil"
-	"github.com/hashicorp/vault/sdk/helper/consts"
 	"github.com/hashicorp/vault/sdk/helper/strutil"
 	"github.com/hashicorp/vault/sdk/logical"
 	"golang.org/x/oauth2"
 )
 
 var oidcStateTimeout = 10 * time.Minute
 
-// OIDC error prefixes. These are searched for specifically by the UI, so any
-// changes to them must be aligned with a UI change.
-const errLoginFailed = "Vault login failed."
-const errNoResponse = "No response from provider."
-const errTokenVerification = "Token verification failed."
+const (
+	// OIDC error prefixes. These are searched for specifically by the UI, so any
+	// changes to them must be aligned with a UI change.
+	errLoginFailed       = "Vault login failed."
+	errNoResponse        = "No response from provider."
+	errTokenVerification = "Token verification failed."
+
+	noCode = "no_code"
+)
 
 // oidcState is created when an authURL is requested. The state identifier is
 // passed throughout the OAuth process.
 type oidcState struct {
 	rolename    string
 	nonce       string
 	redirectURI string
+	code        string
+	id_token    string
```

Comment:
Needs to be refactored to camel case vs. snake case.

Reply:
This is addressed in https://github.com/hashicorp/vault-plugin-auth-jwt/pull/99 if we merge it.